muckrock/accounts/models.py

```diff
@@ -30,6 +30,7 @@
 from muckrock.accounts.querysets import ProfileQuerySet
 from muckrock.core.utils import cache_get_or_set, squarelet_get, stripe_retry_on_error
 from muckrock.organization.models import Organization
+from muckrock.foia.models import FOIAFile
```

Comment:
This import is causing a circular import - can you move it into the `sum_pages_for_user` method?  (You should be able to verify this locally by running the existing tests)